app/decorators/video_decorator.rb

```diff
@@ -0,0 +1,12 @@
+class VideoDecorator < Draper::Decorator
+  delegate_all
+
+  def show_rating
+    average_rating ? "#{show_average_rating}/5.0" : "N/A"
+  end
+
+  private
+  def show_average_rating
```

Comment:
I'm not sure I would use `show_` here since this method does not show anything, it is just used by a method which does.